vendor/processors/stripe_processor.py

```diff
@@ -462,29 +470,41 @@ def create_stripe_customers(self, customers):
         for profile in customers:
             profile_data = self.build_customer(profile)
             new_stripe_customer = self.stripe_create_object(self.stripe.Customer, profile_data)
-
+            print(f'building new customer off {profile_data}')
             if new_stripe_customer:
                 profile.meta['stripe_id'] = new_stripe_customer['id']
                 profile.save()
 
     def update_stripe_customers(self, customers):
         for profile in customers:
-            customer_id = profile.meta['stripe_id']
-            profile_data = self.build_customer(profile)
-            existing_stripe_customer = self.stripe_update_object(self.stripe.Customer, customer_id, profile_data)
+            customer_id = profile.meta.get('stripe_id')
+            print(f'checking on customer {profile} and meta {profile.meta}')
+            if customer_id:
+                profile_data = self.build_customer(profile)
+                print(f'updating new customer off {profile_data}')
+                existing_stripe_customer = self.stripe_update_object(self.stripe.Customer, customer_id, profile_data)
 
-            if existing_stripe_customer:
-                profile.meta['stripe_id'] = existing_stripe_customer['id']
-                profile.save()
+                if existing_stripe_customer:
+                    profile.meta['stripe_id'] = existing_stripe_customer['id']
+                    profile.save()
 
     def sync_customers(self, site):
         stripe_customers = self.get_stripe_customers(site)
-        stripe_customers_emails = [customer_obj['email'] for customer_obj in stripe_customers]
-        vendor_customers_in_stripe = self.get_vendor_customers_in_stripe(stripe_customers_emails)
-        vendor_customers_not_in_stripe = self.get_vendor_customers_not_in_stripe(stripe_customers_emails)
+        stripe_customers_emails = []
+
+        if stripe_customers:
```

Comment:
You can remove that empty var init `stripe_customers_emails` and remove that `if` when you change the `def get_stripe_customers()` function to return an empty list instead of None